addons/crm/models/crm_team.py

```diff
@@ -25,6 +25,19 @@ class Team(models.Model):
         compute='_compute_opportunities',
         string='Amount of quotations to invoice', readonly=True)
     dashboard_graph_model = fields.Selection(selection_add=[('pipeline', 'Pipeline')])
+    dashboard_graph_period_pipeline = fields.Selection([
```

Comment:
As we agree on the purpose but not on how to do it, I let you choose between your solution that will populate the world with evil kittens from hell with laser beams and the other one that will populate the world with walking pizzas.